website/content/en/preview/concepts/deprovisioning.md

```diff
@@ -185,14 +185,6 @@ __Behavioral Fields__
 
 To enable the drift feature flag, refer to the [Settings Feature Gates]({{<ref "./settings#feature-gates" >}}).
 
-Karpenter will annotate the nodes with the `karpenter.sh/voluntary-disruption: "drifted"` if the node is drifted, and does not have the annotation,
-
-Karpenter will remove the  `karpenter.sh/voluntary-disruption: "drifted"` annotation for the following these scenarios:
```

Comment:
We do a status condition now instead. Can you add replace this information with the same information about the status condition?